### PR TITLE
Auth specs

### PR DESCRIPTION
**Pull request by cdmo**
for #19

Specs added for controllers, stubs for views and models due to POC status

## Additional changes

* Corrected attr accessor to use snake case
* Updated rails-rspec to 4.0.0.beta4

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -25,15 +25,13 @@ group :development do
 end
 
 group :development, :test do
-  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'byebug'
   gem 'niftany'
   gem 'pry-byebug'
   gem 'rails-controller-testing'
-  gem 'rspec-rails'
+  gem 'rspec-rails', '~> 4.0.0.beta1'
```
Comment (banukutlu) [inline Gemfile]: for rails 6 👍

File: spec/views/session/index.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'session/index', type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (banukutlu) [inline spec/views/session/index.html.erb_spec.rb]: @cdmo  shouldn't the directory be `spec/views/sessions/`?

Comment (cdmo) [reply spec/views/session/index.html.erb_spec.rb]: :nods:

File: spec/controllers/summaries_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        fields: {
+          circRecordList: [{ key: 1 }]
+        }
+      }.with_indifferent_access
+    end
+
+    before do
+      allow(controller).to receive(:current_user).and_return(user)
+      allow(mock_client).to receive(:patron_info).with(user, item_details: {}).and_return(mock_response)
+      # allow(mock_app_controller).to receive(:current_user).and_return(user)
```
Comment (banukutlu) [inline spec/controllers/summaries_controller_spec.rb]:
```suggestion
```

Comment (banukutlu) [reply spec/controllers/summaries_controller_spec.rb]: @cdmo ☝️ ?

File: spec/controllers/summaries_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SummariesController do
+  let(:mock_client) { instance_double(SymphonyClient) }
+  let(:mock_app_controller) { instance_double(ApplicationController) }
```
Comment (banukutlu) [inline spec/controllers/summaries_controller_spec.rb]:
```suggestion
```

Comment (banukutlu) [reply spec/controllers/summaries_controller_spec.rb]: @cdmo also this?

File: spec/controllers/summaries_controller_spec.rb (hunk 3 above)
Comment (banukutlu) [inline spec/controllers/summaries_controller_spec.rb]: and this?

**Pull request merged**